it/pom.xml

```diff
@@ -211,4 +218,13 @@
     </plugins>
   </build>
 
+  <profiles>
+    <profile>
+      <id>apple-silicon</id>
+      <properties>
+        <os.detected.classifier>osx-x86_64</os.detected.classifier>
+      </properties>
+    </profile>
+  </profiles>
+
```

Comment:
Not sure if there's any impact for non-M1. What I have done is put the profiles on my own `~/.m2/settings.xml`, so at least we don't need to push anything:
```
<settings>
  <activeProfiles>
    <activeProfile>
      apple-silicon
    </activeProfile>
  </activeProfiles>
  <profiles>
    <profile>
      <id>apple-silicon</id>
      <properties>
        <os.detected.classifier>osx-x86_64</os.detected.classifier>
      </properties>
    </profile>
  </profiles>
</settings>
```